be/src/runtime/exec_env.cpp

```diff
@@ -470,6 +472,14 @@ Status ExecEnv::init(const std::vector<StorePath>& store_paths, bool as_cn) {
                     .build(&load_segment_pool));
     _load_segment_thread_pool = load_segment_pool.release();
 
+    RETURN_IF_ERROR(ThreadPoolBuilder("update_tablet_meta_pool")
+                            .set_min_threads(0)
+                            .set_max_threads(config::max_update_tablet_meta_threads)
+                            .set_max_queue_size(config::load_segment_thread_pool_queue_size)
+                            .set_idle_timeout(MonoDelta::FromMilliseconds(10000))
+                            .build(&update_tablet_meta_pool));
+    _update_tablet_meta_pool = update_tablet_meta_pool.release();
+
```

Comment:
Could reuse _thread_pool_alter_tablet, tool many thread pools and thread pool config items are confused ?